### ISSUE TITLE
[↨View+] Loading of trajectories

### ISSUE THREAD

**Issue by hmoenck** [BioroboticsLab/biotracker_core#164]
Loading trajectories does not work in the View+ branch. The trajectory doesn't show up and the biotracker might crash.

**Comment by failure-to-thrive** [comment]
The culprit seems to be there https://github.com/BioroboticsLab/biotracker_core/commit/a0d040246cc5347c16b77db64eb5cdc123c6e9ef#diff-6543cce38ebba4c11a3fc7286296c1b2L583
I can advocate neither `trajectory->getLastChild()` nor `trajectory->getChild(m_currentFrameNumber)` nor anything else without deepest knowledge of project's internals. Just shared my findings. Hope that helps.

**Comment by failure-to-thrive** [comment]
20a07b59f345af6bfe7c4dd4a88e02b51ba25e28 fixed it on **master**.

**Comment by hmoenck** [comment]
No, that refers to an earlier problem with identical symptoms on the master branch. This refers to an issue introduced with the new features in View+. I just fixed it in 369335daa60846b7773f80d89ee9116197dbd4a9.
Long story short: A datafield which is mandatory was not set per default and wasn't set by the deserializer either. It is now set per default, as view properties are not serialized.
